### PR TITLE
Docs: Add permission validation enabled by default to what's new

### PR DESCRIPTION
**Pull request by mgyongyosi**
**What is this feature?**

Update what's new docs with 10.2 changes.

**Why do we need this feature?**

**Who is this feature for?**

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -74,3 +74,12 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
...
 {{% /admonition %}}
+
+
+## Permission validation on custom role creation and update
+
+<!-- Mihaly Gyongyosi -->
+
+_Generally available in Grafana Enterprise_
+
+With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during a custom role creation or update are validated against the list of [available permissions and their scopes]({{< relref "../administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions" >}}). If the request contains a permission that is not available or the scope of the permission is not valid, the request will be rejected with an error message.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:85]: You did the right thing with the relref, we're just updating to a simplified linking format.
```suggestion
With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during a custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.  
```

**Review by imatwawana — APPROVED** [review]
Thanks for contributing! This looks good apart from a minor edit!

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -74,3 +74,12 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
+
+
+## Permission validation on custom role creation and update
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:79]: Wondering if this should be an H3 under an Authorization and authentication H2?
```suggestion
## Permission validation on custom role creation and update
```

Comment (mgyongyosi) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: I moved this and the refresh token part to the `Authentication and Authorization` section

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 3]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
 
 Learn how to upload images here: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets
 -->
+
+## Authentication and Authorization
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:43]:
```suggestion
## Authentication and authorization

```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 4]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+### Configure refresh token handling separately for OAuth providers
+
+<!-- Mihaly Gyongyosi -->
+
+_Available in public preview in all editions of Grafana_
+
+With Grafana v9.3, we introduced a feature toggle called `accessTokenExpirationCheck`. It improves the security of Grafana by checking the expiration of the access token and automatically refreshing the expired access token when the user is logged in using one of the OAuth providers.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:50]:
```suggestion
With Grafana v9.3, we introduced a feature toggle called `accessTokenExpirationCheck`. It improves the security of Grafana by checking the expiration of the access token and automatically refreshing the expired access token when a user is logged in using one of the OAuth providers.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 5]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+
+With the current release, we introduce a new configuration option for each OAuth provider called `use_refresh_token` that allows you to configure whether the particular OAuth integration should use refresh tokens to automatically refresh access tokens when they expire. In addition, to further improve security and provide secure defaults, `use_refresh_token` is enabled by default for providers that support either refreshing tokens automatically or client-controlled fetching of refresh tokens. It's enabled by default for the following OAuth providers: `AzureAD`, `GitLab`, `Google`.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:52]:
```suggestion
With the current release, we've introduced a new configuration option for each OAuth provider called `use_refresh_token` that allows you to configure whether the particular OAuth integration should use refresh tokens to automatically refresh access tokens when they expire. In addition, to further improve security and provide secure defaults, `use_refresh_token` is enabled by default for providers that support either refreshing tokens automatically or client-controlled fetching of refresh tokens. It's enabled by default for the following OAuth providers: `AzureAD`, `GitLab`, `Google`.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 6]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+
+For more information on how to set up refresh token handling, please refer to [the documentation of the particular OAuth provider.]({{< relref "../setup-grafana/configure-security/configure-authentication/" >}}).
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:54]: You were right to use a relref, but we're moving to a simplified linking style, so just reformatting this.
```suggestion
For more information on how to set up refresh token handling, please refer to [the documentation of the particular OAuth provider.](https://grafana.com/docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-security/configure-authentication/).
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 7]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+
+{{% admonition type="note" %}}
+The `use_refresh_token` configuration must be used in conjunction with the `accessTokenExpirationCheck` feature toggle. If you disable the `accessTokenExpirationCheck` feature toggle, Grafana will not check the expiration of the access token and will not automatically refresh the expired access token, even if the `use_refresh_token` configuration is set to `true`.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:57]:
```suggestion
The `use_refresh_token` configuration must be used in conjunction with the `accessTokenExpirationCheck` feature toggle. If you disable the `accessTokenExpirationCheck` feature toggle, Grafana won't check the expiration of the access token and won't automatically refresh the expired access token, even if the `use_refresh_token` configuration is set to `true`.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 8]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
+{{% /admonition %}}
+
+### Permission validation on custom role creation and update
+
+<!-- Mihaly Gyongyosi -->
+
+_Generally available in Grafana Enterprise_
+
+With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during a custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:68]:
```suggestion
With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.  
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 9]
```diff
@@ -39,3 +39,30 @@ You must use relative references when linking to docs within the Grafana repo. P
 
 Learn how to upload images here: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets
 -->
+
+## Authentication and Authorization
+### Configure refresh token handling separately for OAuth providers
+
+<!-- Mihaly Gyongyosi -->
+
+_Available in public preview in all editions of Grafana_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:48]:
```suggestion
_Available in public preview in all Grafana Open Source and Enterprise_
```

**Review by imatwawana — APPROVED** [review]
Thank you for contributing! Apart from some styling and wording tweaks, this is good to go. Because the first feature is available in all editions of Grafana, I'll be adding it to the Cloud What's new as well and I'll tag you for review on that PR shortly.

**Pull request merged**